lib/std/math.zig

```diff
@@ -1324,6 +1325,7 @@ test "lossyCast" {
     try testing.expect(lossyCast(u32, @as(i16, -255)) == @as(u32, 0));
     try testing.expect(lossyCast(i9, @as(u32, 200)) == @as(i9, 200));
     try testing.expect(lossyCast(u32, @as(f32, maxInt(u32))) == maxInt(u32));
+    try testing.expect(lossyCast(u32, nan(u8)) == 0);
```

Comment:
Still the same issue as before with `nan(u8)`.
I think you meant to write `nan(f32)` or some other float.

Reply:
Good catch, I didn't see that.